example_configs/radicale.md

```diff
@@ -0,0 +1,27 @@
+#Configuration of RADICALE authentification with lldap.
+
+# See how to implement the correct plugin (v0.3) for radicale : https://github.com/shroomify-it/radicale-auth-ldap-plugin
+
+# Full docker-compose stack : https://github.com/shroomify-it/docker-deploy_radicale-agendav-lldap
+
+[auth]
```

Comment:
That goes into a configuration file. Which one?

Also, you can surround the whole config with triple backticks ` to make it a code block. Best is if you start with ```toml